.env.sample

```
@@ -5,6 +5,9 @@ DATABASE_URL=postgresql://localhost/hackerspub
 KV_URL=file:///tmp/kv.db
 SECRET_KEY=  # openssl rand -hex 32
 INSTANCE_ACTOR_KEY=  # mise run keygen
+WEB_PUSH_VAPID_PRIVATE_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)